src/app/store/controller/selectors.ts

```diff
@@ -373,6 +374,34 @@ const getByFabricId = createSelector(
       }, [])
 );
 
+/**
+ * Get all region controllers and separate by their vault configuration status.
```

Comment:
I'd personally break this down into 2 selectors, would make it easier to understand and make the comment mostly redundant.
e.g. `vaultConfiguredControllers` `vaultUnconfiguredControllers`

otherwise maybe something like `getByValueConfigurationStatus` returning `{ configured, unconfigured }`.

Reply:
Just to save time on implementation I'm gonna go with wrapping the two in an object, since these two return values are always needed together in their implementations